pom.xml

```diff
@@ -92,6 +92,7 @@
         <netty-all.version>4.1.17.Final</netty-all.version>
         <spark.warehouse>-Dsplice.spark.sql.warehouse.dir=${project.build.directory}/spark-warehouse</spark.warehouse>
         <hbase.rootdir>-Dhbase.rootdir=${project.build.directory}/hbase</hbase.rootdir>
+        <guava.version>14.0.1</guava.version>
```

Comment:
This version affects all profiles.
If guava is provided its version must match the one available at runtime.
For CDH 6.3.2, HBase is run with guava-11.0.2.jar while Spark loads two versions: guava-11.0.2.jar and guava-14.0.1.jar.
Two potential problems may occur: 
* Splice code running in HBase doesn't find a new method/field only available in 14.0.1
* deserialization in Spark fails to match a class due to version difference between HBase and Spark